apollo-router/src/plugins/cache/snapshots/apollo_router__plugins__cache__tests__no_cache_control-4.snap

```diff
@@ -5,5 +5,15 @@ expression: response
 {
   "data": {
     "currentUser": null
-  }
+  },
+  "errors": [
+    {
+      "message": "subgraph mock not configured",
+      "path": [],
+      "extensions": {
+        "code": "SUBGRAPH_MOCK_NOT_CONFIGURED",
+        "service": "user"
+      }
+    }
+  ]
```

Comment:
@SimonSapin Is it intentional?
From the code, it looks like snapshots should stay the same 🤔

Reply:
Yes, this new error comes from the commit [TestHarness subgraph intercept: proper plugin that returns errors](https://github.com/apollographql/router/pull/7060/commits/f6de468a23aa0ed0b953116721f9ff78c7599a1a). Before that, `TestHarness::subgraph_network_requests == false` would cause a subgraph hook to respond with an empty JSON object

Reply:
In this case, I don't understand why we have an error originating in the mocking code inside the snapshot of cache tests.
Is it an issue with the test that had this problem but before this change making it didn't trigger any errors?
And you planning to fix those tests in the follow up PR?
Or am I totally missing context here, and is it normal for this error to be inside a snapshot?